UKHomeOffice/cerberus-service
# COP-12171 - Tis Autopopulation

## Description
This PR plugs up a few more gaps in the auto population logic of the TIS form (mainly after a target is issued and then withdrawn and the form is again auto-populated with new data).

## To Test
- Pull and run against dev
- Load up dev and pick a task
- Spin up the UI locally and load up the same task (ideally best to have both in separate windows).
- Issue a target
- Withdraw it
- Then click on issue target.
- Compare both versions of the re-populated target information sheet (most fields should now be repopulated).
